examples/NetCore2/ConsoleExample/Program.cs

```diff
@@ -28,6 +28,7 @@ private static IServiceProvider BuildDi()
 
             services.AddSingleton<ILoggerFactory, LoggerFactory>();
             services.AddSingleton(typeof(ILogger<>), typeof(Logger<>));
+            services.AddLogging((builder) => builder.SetMinimumLevel(LogLevel.Trace));
```

Comment:
Hey @304NotModified ,

Why do we have to call SetMinimumLevel(Loglevel.Trace) to configure the microsoft log level?  Why doesnt it just utilize the configuration in our nlog.config?

Reply:
good question. That's indeed better if this step isn't needed.